src/prefect_collection_registry/cli.py

```diff
@@ -0,0 +1,54 @@
+import asyncio
+import os
+import sys
+from typing import NoReturn
+from uuid import UUID
+
+from prefect import get_client
+from prefect.blocks.system import Secret
+from prefect.context import FlowRunContext
+from prefect.results import ResultStore
+from prefect.task_runners import ThreadPoolTaskRunner
+
+from prefect_collection_registry.update_collection_metadata import (
+    update_all_collections,
+    update_collection_metadata,
+)
+
+
+async def run_single_collection(collection_name: str, branch_name: str) -> None:
+    """Run metadata update for a single collection."""
+    await update_collection_metadata(collection_name, branch_name)
+
+
+async def run_all_collections(branch_name: str = "update-metadata") -> None:
+    """Run metadata update for all collections."""
+    await update_all_collections(branch_name)
+
+
+def main() -> NoReturn:
+    """CLI entrypoint."""
+    os.environ["GITHUB_TOKEN"] = Secret.load("gh-util-token", _sync=True).get()  # type: ignore
+    if len(sys.argv) == 1:
+        # No args - run all collections
+        asyncio.run(run_all_collections())
+        sys.exit(0)
+    elif len(sys.argv) == 4:
+        # Three args - run single collection
+        collection_name, branch_name, flow_run_id = sys.argv[1:]
```

Comment:
`flow_run_id` isn't documented in the usage string. What's the case where you would pass in a flow run ID?

Reply:
when i want the subprocess to be recognized as a part of the parent flow run